corral/main.pony

```diff
@@ -48,5 +50,5 @@ actor Main
 
     // Hand off to Executor to resolve required dirs and execute the command
     Executor.execute(
-      command, env, log, uout,
+      command._1, env, log, command._2,
```

Comment:
Not sure how elegant this is in Pony vernacular. Seems like it might be worth refactoring the `Executor` in the future, but this felt like the lower friction move for now. Happy to adjust if you prefer another pattern :+1: